lib/std/Build/Step/CheckObject.zig

```diff
@@ -1,4 +1,4 @@
-const std = @import("../std.zig");
+const std = @import("std");
```

Comment:
Interesting.  Should files in the `std` module always be importing std by alias like this now?  If so what's the reason(s)?  I'm curious if non-std modules should be adopting the same convention.

Reply:
Just to note in order to do this type of thing in non-std modules you have to add the module to its own dependencies: https://github.com/ziglang/zig/issues/14708#issuecomment-1440770273

Reply:
I was wondering if we could avoid relative path on std import. It seems yes, we are already doing this in `lib/std/Build/Cache.zig` https://github.com/ziglang/zig/blob/5df31f3ef3d2c4640fa2c7fc9b03c83c6a8606ae/lib/std/Build/Cache.zig#L76